Css/styles1.css

```diff
@@ -1,72 +1,175 @@
+html *{
```

Comment:
This shouldn't be applied to *, because it affects each single element, ideally if you want to add spacing to the whole content, use the body selector